Back/main.py

```diff
@@ -1,13 +0,0 @@
-from fastapi import FastAPI
-from Endpoints.hello import hello as hello_from_api
-
-app = FastAPI()
-
-@app.get("/")
-def hello():
-    return hello_from_api()
-
-if __name__ == "__main__":
-    import uvicorn
-    uvicorn.run(app, host="localhost", port=8082)
```

Comment:
isn't this mandatory?

Reply:
It work on my machine

Reply:
Seriousely, this is replace by the command in the dockerfile